plat/allwinner/common/include/platform_def.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2017-2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __PLATFORM_DEF_H__
+#define __PLATFORM_DEF_H__
+
+#include <common_def.h>
+#include <sunxi_mmap.h>
+#include <tbbr/tbbr_img_def.h>
+
+#define BL31_BASE			SUNXI_SRAM_A2_BASE
+#define BL31_LIMIT			(SUNXI_SRAM_A2_BASE + SUNXI_SRAM_A2_SIZE)
```

Comment:
I don't see why it would be troublesome to change this at any time later? For *now* I wouldn't recommend to use this port anyway, and we would need to change U-Boot for that, which is some bigger problem to do consistently, I guess. And adding some arisc (Allwinner's term for SCP) firmware would require changing the FIT image generation (script) in U-Boot anyway.

Reply:
True, we'd have to change the entry point at the same time as adding the blob anyway, so it's fine to leave things the way they are now.